digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/DynamicLinearAssetService.scala

```diff
@@ -169,7 +170,7 @@ class DynamicLinearAssetService(roadLinkServiceImpl: RoadLinkService, eventBusIm
     }
   }
 
-  private def validateRequiredProperties(typeId: Int, properties: Seq[DynamicProperty]): Unit = {
+  def validateRequiredProperties(typeId: Int, properties: Seq[DynamicProperty]): Unit = {
```

Comment:
Is there any reason to remove private?

Reply:
Yes, to be used on PavedRoadService, since now this service extends from the DynamicLinearAssetService and we need to execute this validation on PavedRoadService too.